sideboard/lib/_threads.py

```diff
@@ -171,25 +171,14 @@ def call(self):
         except Empty:
             pass
 
-    def start(self):
-        self.q.task.start()
-        DaemonTask.start(self)
-
-    def stop(self):
-        self.q.task.stop()
-        DaemonTask.stop(self)
-
     def defer(self, *args, **kwargs):
         self.q.put([args, kwargs])
 
-    def delayed(self, delay, *args, **kwargs):
-        self.q.put([args, kwargs], delay=delay)
```

Comment:
As mentioned above, this is technically a backwards-incompatible change in that we've removed the ``delayed`` parameter, but as mentioned above this method was never actually used by anyone.